src/create-render-context.js

```diff
@@ -23,54 +24,41 @@ class CustomResourceLoader extends jsdom.ResourceLoader {
         this._active = false;
     }
 
-    _executeFromCache(url) {
-        const requestStats = {};
-
-        return fetchPackage(url, requestStats)
-            .then(script => {
-                const source = requestStats.fromCache != null ? "CACHE" : "FETCH";
-                const message = `${source}: ${url}`;
-
-                if (this._active) {
-                    // eslint-disable-next-line no-console
-                    console.log(`${message}`);
-                    this._runScript(script);
-                } else {
-                    // eslint-disable-next-line no-console
-                    console.warn(`File requested but never used (${message})`);
-                }
-            }).then(() => {
-                // Our JS queue will handle executing the code, so we just
-                // resolve with a dummy script.
-                return this.EMPTY;
-            });
+    _fetchJavaScript(url) {
+        return fetchPackage(url).then(({content}) => {
+            if (!this._active) {
+                logging.silly(`File requested but never used (${url})`);
+                return Promise.resolve(this.EMPTY);
+            }
+            return Promise.resolve(new Buffer(content));
+        });
     }
 
     fetch(url, options) {
```

Comment:
Is this `options` argument needed? It seems that we're not using it inside this method.

Reply:
I had it because it was part of the CustomResourceLoader API. It felt handy to keep it so that future maintainers knew it existed. I could lean either way.